api/.eslintrc

```diff
@@ -19,6 +19,7 @@
     "Application": true,
     "ModNote": true,
     "Modmail": true,
-    "Modmails": true
+    "Modmails": true,
+    "_": true
```

Comment:
It's provided as a global by sails anyway, but we had never actually used it so we didn't tell eslint about it until now.